ingestion/src/metadata/ingestion/source/bigquery.py

```diff
@@ -175,6 +175,8 @@ def fetch_sample_data(self, schema: str, table: str) -> Optional[TableData]:
                 return TableData(columns=cols, rows=rows)
             except Exception as err:
                 logger.error(err)
```

Comment:
Returning None in exception should be more efficient, if we add this, we won't need else condition, just calling super outside if condition should be enough

Reply:
yep, feel free to short-circuit a function execution in these cases. running pylint on this code might help you pick up these kinds of improvements!